buildSrc/src/main/kotlin/io/spine/internal/gradle/javadoc/JavadocConfig.kt

```diff
@@ -49,15 +50,25 @@ object JavadocConfig {
     val encoding = Encoding("UTF-8")
 
     fun applyTo(project: Project) {
-        val docletOptions = project.tasks.javadocTask().options as StandardJavadocDocletOptions
+        val javadocTask = project.tasks.javadocTask()
+        configureTask(javadocTask)
+        val docletOptions = javadocTask.options as StandardJavadocDocletOptions
+        configureDoclet(docletOptions)
+    }
+
+    private fun configureTask(javadocTask: Javadoc) {
```

Comment:
Why do we need this redirection?

Reply:
1. `configureDoclet(docletOptions)` is required to group specific actions onto `docletOption`. 
2. `configureTask(javadocTask: Javadoc)` is not required to group actions because only a single one is present.

The latter exists because I didn't want to mix those "specific action" and "group of specific actions" in the same method. IMHO, it just looks better.

```
        val javadocTask = project.tasks.javadocTask()
        configureTask(javadocTask)
        val docletOptions = javadocTask.options as StandardJavadocDocletOptions
        configureDoclet(docletOptions)
```

VS

```
        val javadocTask = project.tasks.javadocTask()
        discardJavaModulesInLinks(javadocTask)(javadocTask)
        val docletOptions = javadocTask.options as StandardJavadocDocletOptions
        configureDoclet(docletOptions)
```

Should I get rid of this?